app/services/analytics.ts

```diff
@@ -0,0 +1,73 @@
+import { action } from '@ember-decorators/object';
+import Service from '@ember/service';
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+export default class Analytics extends Service {
+    metrics = Ember.inject.service();
+    session = Ember.inject.service();
+
+    @action
+    click(category, label, extraInfo) {
+        let extra = extraInfo;
+        if (extra && typeof extra !== 'string') {
```

Comment:
Maybe add a comment here about how this is to remove the event object when used with `onclick`?

Reply:
This is all copy-pasted from the mixin, but sure.

Reply:
Opportunity to improve it! :wink: